.travis.yml

```diff
@@ -21,10 +23,11 @@ addons:
 sudo: false
 before_install:
   - cpanm --local-lib=~/perl5 local::lib && eval $(perl -I ~/perl5/lib/perl5/ -Mlocal::lib)
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH="$HOME/.yarn/bin:$PATH"
 install:
   - cpanm --quiet --installdeps --notest --skip-satisfied .
-  - npm install
```

Comment:
Shouldn't `npm install` stay because of the jshint dependency?

Reply:
`yarn` is auto-detected by Travis and install `jshint` by reading `package.json`
See the end of the success build logs at https://travis-ci.org/nicolasleger/openfoodfacts-server